feature-group-definitions/inert.yml

```
@@ -0,0 +1,7 @@
+name: Inert subtrees
```

```suggestion
name: Inert elements
```

"Subtrees" is a metaphor we're not going to explain and I don't know if it's self-evident. Since docs more often talk about parents and children, I'm suggesting to sidestep this (with a little help from the description).

Alternatively we could do like `autofocus` though this feels a little too terse to me:

```suggestion
name: Inert
```

---

I think "`inert` attribute" or "Inert elements" are both OK. https://wpt.fyi/interop-2023 did call this simply "Inert" but I agree it's unnecessarily terse, in that case I'd want it to be `inert` with code syntax to point at the attribute.